### ISSUE TITLE
Advertise evalcast, modeltools

### ISSUE THREAD

**Issue by ryantibs** [cmu-delphi/covidcast#396]
We should create a pkgdown site for evalcast with a nice vignette; if this exists, then I don't see it on the main branch.

And we should advertise both evalcast & modeltools from the "front page" (README) of this covidcast repo, say right under the "API Clients" section.

Here is the [pkgdown site for modeltools](https://cmu-delphi.github.io/covidcast/modeltoolsR/) that I just built.  The vignettes are pretty polished, except the last one doesn't have polished text at all (oh well).

**Comment by huisaddison** [comment]
Hey @dajmcdon , I noticed this issue and #355 were closed.  Is this because of plans to restructure the packages into separate repositories?  Since I noticed we haven't linked to evalcast vignettes in the `covidcast` repository [README](https://github.com/cmu-delphi/covidcast/).  In fact, the evalcast vignettes (and documentation) are not in the repository in compiled form, so navigating to the expected webpage gives a 404 error: https://cmu-delphi.github.io/covidcast/evalcastR/
